packages/@aws-cdk/cdk/lib/dependency.ts

```diff
@@ -27,17 +24,61 @@ export interface IDependable {
 export class ConcreteDependable implements IDependable {
```

Comment:
Where is this used?

Reply:
https://github.com/awslabs/aws-cdk/search?q=concretedependable&unscoped_q=concretedependable

Reply:
Oh nice. for some reason I thought it is just introduced now.